plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java

```diff
@@ -467,6 +469,29 @@ private synchronized void doAppendError(String s) {
     }
   }
 
+  private synchronized void doFlush() {
```

Comment:
well, philosophically speaking functions compute (without side effects) and other do something with side effects - this has side effects. :-)